spec/models/user_spec.rb

```diff
@@ -302,6 +159,19 @@
         to raise_error(ActiveRecord::RecordInvalid,
                        /Validation failed: Password is too long \(maximum is 128 characters\)/)
     end
+
+    it 'works with spaces' do
+      PASSWORD_W_SPACES = 'this has a few spaces'.freeze
+
+      prototype_user = create(:user)
+
+      # Verify success.
+      user = create(:user,
+                    email: "mkuniq.#{prototype_user.email}",
+                    password: PASSWORD_W_SPACES,
+                    password_confirmation: PASSWORD_W_SPACES)
+      expect(user.errors.any?).to be_falsey
```

Comment:
In general `be_falsey` and `be_truthy` are not recommended. Here's a philosophical discussion if you're interested: https://github.com/rspec/rspec-expectations/issues/283

I would suggest `expect(user.errors).to be_empty` or `expect(user.errors.size).to eq 0`